org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/CompletionHandlerTest.java

```diff
@@ -319,6 +320,30 @@ public void testCompletion_import_package() throws JavaModelException{
 		//Not checking the range end character
 	}
 
+	@Test
+	public void testCompletion_javadocComment() throws JavaModelException {
```

Comment:
maybe add some more tests for:
- already existing javadoc 
- partially existing javadoc (has only 1 `@param` doc'ed)
- no completion after regular comment /* */
- no completion for no-param methods